lib/recipes-data/src/lib/eventbus.ts

```diff
@@ -0,0 +1,73 @@
+import * as process from "process";
+import {EventBridgeClient,PutEventsCommand} from "@aws-sdk/client-eventbridge";
+import {registerMetric} from "@recipes-api/cwmetrics";
+import {OutgoingEventBus} from "./config";
+import type { RecipeIndexEntry, RecipeReference } from './models';
+
+const ebClient = new EventBridgeClient({region: process.env["AWS_REGION"]});
+
+export async function announce_new_recipe(updated:RecipeReference[], removedList:RecipeIndexEntry[]) {
+  if(!OutgoingEventBus || OutgoingEventBus =="") {
```

Comment:
Should this be failing at the point we extract the config, rather than here? `mandatoryParameter` exists for this reason, I think.

Reply:
These are here because event bus was not a mandatory parameter - the responder still works even if eventbus doesn't so we don't break publication (error thrown here is caught and non-fatal). Can change in a future PR but I'd rather keep as-is right now